test/signature-unit-tests.spec.ts

```diff
@@ -1265,4 +1265,14 @@ describe("Signature unit tests", function () {
       "MIIDZ",
     );
   });
+
+  it("can decrypt the private key when privateKeyPass is set", function () {
```

Comment:
I would be more thorough to also have a test that does throw when trying to use this key when the passphrase isn't provided; please add that.